os/include/tinyara/signature.h

```diff
@@ -22,9 +22,9 @@
 #include <tinyara/config.h>
 #include <stdint.h>
 
-#if defined(CONFIG_APP_BINARY_SEPARATION) && defined(CONFIG_ARCH_BOARD_RTL8721CSM)
+#if defined(CONFIG_APP_BINARY_SEPARATION) && defined(CONFIG_ARCH_BOARD_RTL8721CSM) && defined(CONFIG_BINARY_SIGNING)
 /* The size of prepended binary signing data */
-#define USER_SIGN_PREPEND_SIZE     0
+#define USER_SIGN_PREPEND_SIZE     32
 #endif
```

Comment:
If this value is board-specific, how about making this as new config instead of this code?